src/data/behandler/behandlereQueryHooks.ts

```diff
@@ -41,3 +42,29 @@ export const useSokBehandlereQuery = (searchstring: string) => {
     data: query.data || [],
   };
 };
+
+export const useBehandlerByBehandlerRefQuery = (
+  behandlerRef: string | false
```

Comment:
`string | false` ser kanskje litt rart ut. 
Tanken er at denne queryen ikke skal kjøres dersom behandlerRef er `false`, noe den vil være dersom man finner behandleren i lista over behandlere for personen. Denne queryen er altså en slags fallback dersom det er behandlere som ikke er tilknyttet personen som har svart (feks behandlere fra søket, eller vikarer som svarer på en melding til en annen behandler).

Reply:
Kan man ikke oppnå det samme om den er null/undefined? Ellers er det vel rart om et annet sted i koden skal kalle på denne hooken, men sende inn et parameter som gjør at den ikke gjør noe likevel? 🤔

Reply:
Jo, det er rart å sende inn noe og så gjør den ikke noe, men det er liksom eneste måten å gjøre kallet conditionally på med ReactQuery slik jeg har forstått det. Vi gjør det samme for å søke etter behandlere ser jeg (den søker ikke hvis man sender inn en tom streng i søket - som sikkert er initialState).

Reply:
Alternativt kunne man droppet denne hooken og heller kalt `useQuery` (med `enabled`-flagg) direkte fra `useBehandlerNavn`, så er det også kanskje mer tydelig at den bare er enabled hvis man ikke finner behandler i lista...

Reply:
Nå bllr det litt dobbelt opp med den andre tråden også, men man må jo ikke bruke enabled i queryen hvis man uansett bare kan la være å kalle på den i utgangspunktet?

Jeg mener det er mer naturlig å gjøre
```
if (skalGjøreTing)
    gjørTing(foo, bar)
```

i stedet for 
```
gjørTing(foo, bar, skalGjøreTing)
```
Da ender man opp med å gjøre `gjørTing(foo, bar, bareTullaIkkeGjørNoeLikevelJegBareKallerPåMetodenForMoroSkyld)`